src/index.js

```diff
@@ -248,8 +230,8 @@ class MarkdownShortcuts {
       for (let match of this.matches) {
         const matchedText = text.match(match.pattern)
         if (matchedText) {
-          console.log('matched', match.name, text)
-          match.action(text, selection, match.pattern, lineStart)
+          console.log('matched', match.name, matchedText[0])
```

Comment:
We don't actually need to replace the whole line because there's maybe user's text inside.
So I replaced it with `matchedText` to calculate length of text we have to delete.